lib/screens/listOfOrgs/widgets.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+Widget appBar() {
```

Comment:
these need to be stateless/stateful widget classes (decided by you)
in their separate files